### ISSUE TITLE
Invalid numeric result

### ISSUE THREAD

**Issue by Merlin6nt** [getredash/redash#6353]
### Issue Summary

When I request the database for a value with a large number, like `select 5949879911342665595` I get wrong result: `5,949,879,911,342,665,728`

![photo_2023-08-07_16-34-05](https://github.com/getredash/redash/assets/43607635/cb2fa1d7-228c-46ee-9335-d6293e99b201)

### Steps to Reproduce

1. Create new query for PG DB or Clickhouse DB.
2. Add query `select 5949879911342665595` and execute

### Technical details:

* Redash Version: 10.1.0 (2589bef1)

**Comment by justinclift** [comment]
Yeah, this is a known problem with very large numbers. :frowning:

@eradman Do you reckon the numbro PR would fix this?  From rough memory, the problem here is in the display side of things as the backend is passing correct values through from the configured data source.

**Comment by eradman** [comment]
Unfortunately the updated library does not fix this. It returns a _different incorrect_ result!

```
> var numbro = require("numbro");
> numbro(5949879911342665595);
t { _value: 5949879911342666000 }
```

And this is a known issue: https://github.com/BenjaminVanRyseghem/numbro/issues/441. I'll ping the upstream ticket.

**Comment by konnectr** [comment]
This is a duplicate https://github.com/getredash/redash/issues/6159

**Comment by eradman** [comment]
Closing in favor of linked ticket
